docs/queries/byrole.mdx

```diff
@@ -62,11 +62,10 @@ oftentimes better to use `getByRole(expectedRole, { name: 'The name' })`. The
 accessible name query does not replace other queries such as `*ByAlt` or
 `*ByTitle`. While the accessible name can be equal to these attributes, it does
 not replace the functionality of these attributes. For example
-`<img aria-label="fancy image" src="fancy.jpg" />` will be returned for both
-`getByAltText('fancy image')` and `getByRole('img', { name: 'fancy image' })`.
```

Comment:
What I did is just removing this "`getByAltText('fancy image')`"

Maybe I could correct `<img aria-label="fancy image" src="fancy.jpg" />` to `<img alt="fancy image" src="fancy.jpg">` as like [this comment](https://github.com/testing-library/testing-library-docs/issues/1190#issuecomment-1355409202). I made this change so that the following sentence makes sense.